targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/device/TOOLCHAIN_ARM_STD/MK66FN2M0xxx18.sct

```diff
@@ -63,12 +65,21 @@
 #define m_interrupts_start             MBED_APP_START
 #define m_interrupts_size              0x00000400
 
+#if MBED_APP_SIZE == 0
```

Comment:
Surely you mean `MBED_APP_START == 0`?

Reply:
Yes. The flash config is needed only if `MBED_APP_START` is 0 since that config has fix address in the HW.